skypy/galaxy/mass.py

```diff
@@ -0,0 +1,97 @@
+r"""Models of galaxy luminosities.
```

Comment:
```suggestion
r"""Models of galaxy mass (total and stellar).
```

If there will not be total galaxy mass in here, then `mass.py` is the wrong name for this file.

Reply:
Not yet. Maybe it comes in the future? I can change the name tho and if there will be total masses at some point, we can change it back.

Reply:
The dark matter masses are handled in the halo module. So this should probably be stellar_mass.py or similar

Reply:
If the subpackage is `stellar_mass` then the function should probably be `schechter`. The other SMF I could think of is `broken_power_law`, so that `galaxy.stellar_mass.{schechter,broken_power_law}` makes more sense than `galaxy.stellar_mass.stellar_mass_function`.

Reply:
Ok this is true. I will change it then to this in the next commit.

Reply:
`schechter` already exists in `utils.random` and I am importing this. This gives an error with flake8 but also it's not really nice to have the same name for e more than one function in SkyPy, right? What do you think about `schechter_mass`?

Reply:
we might not need the Schechter function in `utils.random` after I push all the code in PR#186. We might need to discuss this offline at some point

Reply:
I'm pretty sure we will want to keep `utils.random.schechter`. You could change how you import it to prevent the collision:
```
from skypy.utils import random
def schechter(...):
    x_sample = random.schechter(...)

Reply:
But is it nice to have more functions called 'schechter'? Also, it might be a bit misleading to call it just 'schechter', cause it's not a schechter function. Although it's in `stellar_mass.py` I don't really like it just calling schechter tbh.

Reply:
You can call it `schechter_smf`, and we can at some point rename the other function to `schechter_lf`. I think that both functions would ideally reside in the same module eventually, since they both deal with the luminous matter of a galaxy (eg. `skypy.galaxy.stars`) and there's little point in fracturing the module hierarchy more than that.